apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -455,6 +568,21 @@ private string GetValueOrDefault(string key, string defaultValue = "")
                 {
                     value = ValidatedDataSource((string)value);
                 }
+                if (string.Equals(keyword, ClrToSpannerTypeDefaultMappingsKeyword, StringComparison.OrdinalIgnoreCase))
```

Comment:
All of these can be `else if`, I think. We could also probably change it just to set the releated property to `(string) value` (and then return), as a way of avoiding duplication.